bin/build-demo.js

```diff
@@ -9,33 +9,24 @@ if (!fs.existsSync('build/loader/loader-debug.js')) {
 
 copy('build/loader/loader-debug.js', 'build/demo/loader-debug.js');
 copy('src/demo/index.html', 'build/demo/index.html');
-copy('src/demo/modules2', 'build/demo/modules2');
-copy('src/demo/modules3', 'build/demo/modules3');
-copy('src/demo/packages', 'build/demo/packages');
+copy('src/demo/config.js', 'build/demo/config.js');
+
+fs.mkdirsSync(`build/demo/resolutions`);
+globby.sync('src/demo/resolutions/*').forEach(file => {
+	const filePath = file.substring('src/demo/resolutions/'.length);
```

Comment:
Maybe?

```suggestion
	const filePath = path.basename(file);
```

I'd probably also change the variable name (`filePath` → `baseName`).